modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -159,6 +179,14 @@ class Stats_Behavioural extends \NDB_Form
             $vl     = $row['VLabel'];
             $c      =& $this->tpl_data['behaviour']['C' . $center];
 
+            if (is_null($c)) {
```

Comment:
remove 182-189
when you get a null value inside this loop, it will throw new Exception on the page, then broke the page.

Reply:
How do you get a centerID value inside the loop? I don't see why that should be possible.

Reply:
@driusan you're right i'll have to look this over again

Reply:
@driusan seems like it works fine